### PR TITLE
Feature/trajectory discussion

### PR DESCRIPTION
**Pull request by helenol**
Depends on https://github.com/ethz-asl/mav_comm_devel/pull/13

### CODE REVIEW

**Comment by helenol** [comment]
test this please

File: rotors_gazebo_plugins/src/gazebo_bag_plugin.cpp [hunk 1]
```diff
@@ -209,6 +213,12 @@ void GazeboBagPlugin::WaypointCallback(
   writeBag(waypoint_topic_, ros_now, trajectory_msg);
 }
 
+void GazeboBagPlugin::CommandPoseCallback(const geometry_msgs::PoseStamped& pose_msg) {
+  common::Time now = world_->GetSimTime();
+  ros::Time ros_now = ros::Time(now.sec, now.nsec);
+  writeBag(waypoint_topic_, ros_now, pose_msg);
```
Comment (burrimi) [inline rotors_gazebo_plugins/src/gazebo_bag_plugin.cpp]: copy paste error `command_pose_topic_`

**Comment by burrimi** [comment]
scolgtm.

File: rotors_gazebo_plugins/src/gazebo_bag_plugin.cpp (hunk 1 above)
Comment (helenol) [inline rotors_gazebo_plugins/src/gazebo_bag_plugin.cpp]: Aaaaaaaah! Great catch, thanks.

**Comment by helenol** [comment]
test this please

**Pull request merged**